### PR TITLE
Add references of u-u Remove-Unused-Kernel-Packages setting

### LINKED ISSUE #79: Please note that unattended-upgrades already removes unused kernels
The unattended-upgrades version found in Xenial and later and Buster and later automatically removes auto-removable kernels (in u-u v1.0 and up).

### PR DESCRIPTION
**Pull request by tonyskapunk**
- Add ref about u-u's `Remove-Unused-Kernel-Packages` setting

Fix #79

---

@rbalint do you mind looking at the changes? Let me know if they read well now. Thanks!

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -28,7 +28,11 @@ By default, on apt 1.0 and below, the booted kernel, the latest-installed kernel
 Ubuntu has multiple suggestions on how to remove kernels:
   https://help.ubuntu.com/community/RemoveOldKernels
 
-A good recommendation is using unattendend-upgrades(`debian <https://wiki.debian.org/UnattendedUpgrades>`__, `ubuntu <https://help.ubuntu.com/community/AutomaticSecurityUpdates>`__) as it has an option to remove unused dependencies, but that applies to all packages not just kernels.
+A great recommendation is to make use of unattended-upgrades `u-u` (`debian <https://wiki.debian.org/UnattendedUpgrades>`__, `ubuntu <https://help.ubuntu.com/community/AutomaticSecurityUpdates>`__). Since version **1.0** (`debian <https://packages.debian.org/search?keywords=unattended-upgrades>`__, `ubuntu <https://packages.ubuntu.com/search?keywords=unattended-upgrades>`__) *u-u* is capable of removing unused kernel packages (*Remove-Unused-Kernel-Packages*).
```
Comment (rbalint) [inline README.rst]: IMO ... is capable of removing unused kernel packages ... -> ... removes unused kernel packages by default ... would be more accurate and easier to understand.

File: README.rst [hunk 2]
```diff
@@ -28,7 +28,11 @@ By default, on apt 1.0 and below, the booted kernel, the latest-installed kernel
...
+Some distributions may not make reference or may comment out that setting in its configuration, but unless it is explicitly disabled, *u-u* will attempt to remove unused kernels. `More info here <https://github.com/rackerlabs/kthresher/issues/79>`__.
```
Comment (rbalint) [inline README.rst]: Also: ... Some distributions .. -> ... Some distribution versions ...

File: README.rst (hunk 1 above)
Comment (tonyskapunk) [reply README.rst]: Fixed in 37f1c64

File: README.rst (hunk 2 above)
Comment (tonyskapunk) [reply README.rst]: Fixed in 37f1c64

**Comment by tonyskapunk** [comment]
@rbalint thanks for the time and feedback, I've made those changes. If everything looks then I'll merge this one.

**Comment by rbalint** [comment]
@tonyskapunk Thanks, looks good now!

**Pull request merged**